### PR TITLE
fix bug in array param in body

### PR DESCRIPTION
**Pull request by sdghchj**
#565

### CODE REVIEW

File: operation_test.go [hunk 1]
```diff
@@ -389,12 +389,16 @@ func TestParseParamCommentBodyArray(t *testing.T) {
 	expected := `{
     "parameters": [
         {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
```
Comment (ubogdan) [inline operation_test.go]: This produce an error in swagger editor.
```
Structural error at paths ./api/something.get.parameters.0
should NOT have additional properties
additionalProperty: type, items
```
Did you want to make the PR more consistent?

Comment (sdghchj) [reply operation_test.go]: Sorry,maybe my gin-swagger is of very old version. I see there is no type displayed below the param name, so I assigned array type to its SimpleSchemaType. Have removed it. A new PR.

**Comment by sdghchj** [comment]
PR also for #581

**Comment by ubogdan** [comment]
@sdghchj I asked you to do a simple change. Code review takes time and i don't have too much lately.

**Comment by sdghchj** [comment]
> @sdghchj I asked you to do a simple change. Code review takes time and i don't have too much lately.

reverted

**Comment by ubogdan** [comment]
Thanks for your contribution !

File: operation_test.go (hunk 1 above)
Comment (easonlin404) [reply operation_test.go]: @ubogdan thank you for point out.

**Pull request merged**